account_bank_statement_import_online/models/online_bank_statement_provider.py

```diff
@@ -394,43 +424,38 @@ def _sanitize_bank_account_number(self, bank_account_number):
         self.ensure_one()
         return sanitize_account_number(bank_account_number)
 
-    @api.multi
-    def _get_next_run_period(self):
-        self.ensure_one()
-        if self.interval_type == "minutes":
-            return relativedelta(minutes=self.interval_number)
-        elif self.interval_type == "hours":
-            return relativedelta(hours=self.interval_number)
-        elif self.interval_type == "days":
-            return relativedelta(days=self.interval_number)
-        elif self.interval_type == "weeks":
-            return relativedelta(weeks=self.interval_number)
-
     @api.model
     def _scheduled_pull(self):
-        _logger.info("Scheduled pull of online bank statements...")
-
-        providers = self.search(
-            [
-                ("active", "=", True),
-                ("next_run", "<=", fields.Datetime.now()),
-            ]
+        _logger.info(_("Scheduled pull of online bank statements..."))
+        today = datetime.today()
+        daystart = datetime(
+            year=today.year,
+            month=today.month,
+            day=today.day,
+            hour=0,
+            second=0
         )
+        providers = self.search([("active", "=", True)])
```

Comment:
Doesn't Odoo do that by default?

Reply:
Yes, but was also already there. Will remove it.